app/src/main/java/com/github/se/gomeet/ui/mainscreens/profile/SettingsScreen.kt

```diff
@@ -0,0 +1,195 @@
+
+              SettingsSubtitle(
+                  "More info and support", Modifier.padding(15.dp).align(Alignment.Start))
+
+              SettingsComposable(R.drawable.baseline_chat_bubble_outline_24, "Help")
+              SettingsComposable(R.drawable.gomeet_icon, "About")
+
+              Text(
+                  text = "Log out",
+                  modifier = Modifier.padding(start = 15.dp),
+                  color = Color.Red,
+                  fontStyle = FontStyle.Normal,
+                  fontWeight = FontWeight.SemiBold,
+                  fontFamily = FontFamily.Default,
+                  textAlign = TextAlign.Start,
+                  style = MaterialTheme.typography.bodySmall)
+
+              Text(
+                  text = "Delete account",
+                  modifier = Modifier.padding(start = 15.dp),
+                  color = Color.Red,
+                  fontStyle = FontStyle.Normal,
+                  fontWeight = FontWeight.SemiBold,
+                  fontFamily = FontFamily.Default,
+                  textAlign = TextAlign.Start,
+                  style = MaterialTheme.typography.bodySmall)
+            }
+      }
+}
+
+@Composable
+fun SettingsComposable(icon: Int, text: String) {
+  Row(
+      modifier = Modifier.padding(start = 15.dp),
+      horizontalArrangement = Arrangement.Start,
+      verticalAlignment = Alignment.CenterVertically) {
+        Icon(
+            painter = painterResource(id = icon),
+            contentDescription = text + "icon",
+            tint = Color.Black)
+
+        Text(
+            text = text,
+            modifier = Modifier.padding(start = 15.dp),
+            color = Color.Black,
+            fontStyle = FontStyle.Normal,
+            fontWeight = FontWeight.SemiBold,
+            fontFamily = FontFamily.Default,
+            textAlign = TextAlign.Start,
+            style = MaterialTheme.typography.bodySmall)
+
+        Spacer(modifier = Modifier.weight(1f))
+
+        NavigateNextComposable()
+      }
+}
+
+@Composable
+fun NavigateNextComposable() {
+  Surface(
+      onClick = { /* Go to the settings more in depth */},
+      modifier = Modifier.padding(end = 15.dp),
+      shape = CircleShape,
+      color = Color.Transparent) {
+        Icon(
+            painter = painterResource(id = R.drawable.navigate_next),
+            contentDescription = "Navigate next icon",
+            modifier = Modifier.padding(15.dp),
+            tint = Color.Black)
+      }
+}
+
+@Composable
+fun SettingsSubtitle(text: String, modifier: Modifier) {
+  Text(
+      text = text,
+      modifier = modifier,
+      color = Color.Gray,
+      fontStyle = FontStyle.Normal,
+      fontWeight = FontWeight.Light,
+      fontFamily = FontFamily.Default,
+      textAlign = TextAlign.Start,
+      style = MaterialTheme.typography.labelSmall)
+}
+
+@Preview
+@Composable
+fun SettingsScreenPreview() {
+  SettingsScreen(nav = NavigationActions(rememberNavController()))
+}
```

Comment:
UI is well-structured and addresses the implementation of a settings screen comprehensively. It is coherent with the Figma. Thank you, Antoine !
The only suggestion made is towards enhancing robustness and responsiveness.